src/components/Modal/Modal.vue

```diff
@@ -728,20 +733,22 @@ $header-size: 50px;
 	&--small {
 		.modal-container {
 			width: 350px;
+			max-height: 90%;
 			height: 450px;
 		}
 	}
 	&--medium {
 		.modal-container {
 			width: 500px;
+			max-height: 90%;
 			height: 600px;
 		}
 	}
 	&--normal {
 		.modal-container {
-			max-width: 70%;
+			max-width: 90%;
```

Comment:
This reduces the size difference between the width of the element when going from over the mobile breakpoint to below